.github/workflows/test-docs.yml

```
@@ -0,0 +1,33 @@
+        with:
+          node-version: "20.11.1"
+      - name: Cache Yarn dependencies
+        uses: actions/cache@v2
+        id: yarn-cache
+        with:
+          path: ~/.cache/yarn
+          key: ${{ runner.os }}-yarn-${{ hashFiles('./packages/logger/yarn.lock') }}
+      - name: Install dependencies
+        run: yarn
+      - name: Check @texturehq/events package version
+        run: >
```

actionlint

shellcheck reported issue in this script: SC1009:info:1:1: The mentioned syntax error was in this variable assignment

(shellcheck)